src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -0,0 +1,109 @@
+from sentry.utils import redis
+
+logger = logging.getLogger(__name__)
+
+# redis key for entry storing current list of LPQ members
+LPQ_MEMBERS_KEY = "fill with appropriate value here"
+# key for cluster that stores list of current LPQ members. may be equivalent to CLUSTER_KEY.
+LPQ_CONFIG_CLUSTER_KEY = "fill with appropriate value here"
+
+# key for cluster used to store metrics for LPQ
+CLUSTER_KEY = "fill with appropriate value here"
+
+
+@instrumented_task(name="sentry.tasks.symbolicator.scan_for_suspect_projects", queue="symbolicator.compute_low_priority_queue", ignore_result=True)  # type: ignore
+def scan_for_suspect_projects() -> None:
+    _scan_for_suspect_projects()
+
+
+def _scan_for_suspect_projects() -> None:
+    # todo: this is probably going to be some class wrapper around the cluster to abstract away most
+    # of the nitty-gritty redis work
+    cluster: redis._RedisCluster = redis.get_cluster_from_options(CLUSTER_KEY)
+
+    suspect_projects = set()
+    for item in cluster.scan_iter(
+        # assuming the format is "symbolicate_event_lpq:<project_id>:<posix timestamp or other>"
+        match="symbolicate_event_lpq:*",
+        count=1000,
+    ):
+        [_key, project_id, _else] = item.split(":")
+        if project_id not in suspect_projects:
+            suspect_projects.add(project_id)
+            calculate_lpq_eligibility(project_id)
+
+    # todo: populate with contents of redis-stored kill switch
+    current_lpq_projects = set()
+    deleted_projects = current_lpq_projects.difference(suspect_projects)
+
+    for deleted in deleted_projects:
+        # todo: may be good to make this a batch delete
+        remove_from_lpq(deleted)
+
+
+@instrumented_task(name="sentry.tasks.symbolicator.calculate_lpq_eligibility", queue="symbolicator.compute_low_priority_queue", ignore_result=True)  # type: ignore
+def calculate_lpq_eligibility(project_id: str) -> None:
+    _calculate_lpq_eligibility(project_id)
+
+
+def _calculate_lpq_eligibility(project_id: str) -> None:
+    # todo: this is probably going to be some class wrapper around the cluster to abstract away most
+    # of the nitty-gritty redis work
+    cluster: redis._RedisCluster = redis.get_cluster_from_options(CLUSTER_KEY)
+    pattern = (f"symbolicate_event_lpq:{project_id}:*",)
+
+    def extract_timestamp(key: str) -> str:
+        [_key, _project_id, timestamp] = key.split(":")
+        return timestamp
+
+    is_eligible = calculation_magic(
+        [extract_timestamp(key) for key in cluster.scan_iter(match=pattern, count=1000)]
+    )
+
+    if not is_eligible:
+        return
+
+    # todo: this is probably going to be some class wrapper around the cluster to abstract away most
+    # of the nitty-gritty redis work
+    cluster: redis._RedisCluster = redis.get_cluster_from_options(LPQ_CONFIG_CLUSTER_KEY)
+    added = cluster.sadd(name=LPQ_MEMBERS_KEY, values=[project_id])
+
+    if added:
+        # cry at sentry
```

Comment:
don't think we need to do anything here?  sentry will read the redis key and that's that

Reply:
iirc we do want to tell sentry something when projects are added and removed from the LPQ list, as a starting step to tracking the general behaviour of these tasks and when/which projects are matching the LPQ criteria. i assume that means sending something over via `capture_message()`, but could i be misinterpreting one of the goals or tasks related to that?

Reply:
oh, that way.  yes we need this probably both as datadog metrics as well as some `capture_message()` probably.  i misunderstood what you meant, the danger of working both on the product and using the product at the same time... 😄